tests/e2e/test_cases_data/mnpt_data.hocon

```diff
@@ -0,0 +1,29 @@
+# test_data.hocon
+# Input/output test pairs
+
+test = [
+  {
+    input_1: {
+      user_text: "Who did yellow submarine?"
+      answer: {
+        type_match: "keyword"
+        word: "Beatles"
+        cost: "3.0"
```

Comment:
That you have cost built in as a key means that this format is very tightly coupled to a particular test.